GlimpseImputationPipeline/glimpse_docker/build_base_and_extension_docker.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env zsh
```

Comment:
Is this required? I don't know if zsh is available on all systems

Reply:
yeah, I'm not sure about this.  when the environment was bash the build was failing in weird ways I couldn't explain and making it zsh fixed the problem.  Probably more an issue with my system than anything else, so I'm fine with switching it back to bash and trying to deal with it on my side.